### DISCUSSION TITLE
Error enrolling YubiKey 5 passkey for 2FA

### DISCUSSION THREAD

**Discussion by hayydeeeeen** [community/community#137892] [Code Security] [answered]
### Select Topic Area

Question

### Body

I recently received an email saying that 2FA will become required for my GitHub account, so I decided to update my 2FA methods, I want to enroll my Yubikey 5 NFC as a passkey, rather than the standard FIDO2 security key option. When I attempt to do so, I receive the error found in the attached screenshot.

![gh_error](https://github.com/user-attachments/assets/8998e6be-97bf-41f9-a020-98a8a38b5d45)

Does anybody have a fix for this, or know how I can get in touch with a human at GitHub

**Comment by Hotanya** [comment]
Hey @hayydeeeeen, do you have any other 2FA methods currently enrolled? Try enrolling another method first and the re-enrol your Yubikey.

**Reply by hayydeeeeen** [reply]
Yes. I have another FIDO2 key (enrolled as security key, not passkey) and an OAUTH app.

**Reply by Hotanya** [reply, marked as answer]
Add the Yubikey normally (standard FIDO2 security key option), once it is enrolled, GitHub automatically recognises it as a passkey and should show up in (https://github.com/settings/security) as a passkey

**Reply by ghostinhershell** [reply]
Hi @hayydeeeeen, did @Hotanya response here answer your question? 

If so, can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2). Thanks!
